packages/fx-core/src/component/resource/botService/common.ts

```diff
@@ -9,3 +10,7 @@ export function genUUID(): string {
 export function isHttpCodeOkOrCreated(code: number): boolean {
   return [200, 201].includes(code);
 }
+
+export function makeBotName(raw: string): string {
+  return raw.length > MaxLengths.BOT_NAME ? raw.substr(raw.length - MaxLengths.BOT_NAME) : raw;
```

Comment:
According to this page https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr. This function is deprecated and it seem we also call it in wrong way.